extensions/in-memory/assetindex-memory/src/main/java/org/eclipse/dataspaceconnector/metadata/memory/InMemoryAssetLoader.java

```diff
@@ -14,14 +14,18 @@
 
 package org.eclipse.dataspaceconnector.metadata.memory;
```

Comment:
I think I missed the point where this was called `InMemoryAssetLoader`, I think the best name for this is still `InMemoryAssetIndex`

Reply:
this might have happened in one of the renaming/refactorings. I'll rename it.